ocs_ci/utility/vsphere.py

```diff
@@ -1512,3 +1512,108 @@ def wait_for_vm_delete(self, vm, timeout=60):
             timeout=timeout, sleep=10, func=self.is_vm_obj_exist, vm=vm
         )
         return sample.wait_for_func_status(result=False)
+
+    def get_network_device(
+        self, ip, dc, label="Network adapter 1", network="VM Network"
+    ):
+        """
+        Get the network adapter for a VM.
+
+        Args:
+            ip (str): The IP address of the VM.
+            dc (str): The datacenter name where the VM is located.
+            label (str, optional): The label of the network adapter. Defaults to "Network adapter 1".
+            network (str, optional): The name of the network. Defaults to "VM Network".
+
+        Returns:
```

Comment:
The ret value here is bool, please change it according to Google Style Python Docstrings